### DISCUSSION TITLE
12/26 멘토링 관련

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#31] [Q&A] [answered]
이번 멘토링때 받고 싶은 내용을 여기에 미리 정리해도 될까요..?

추가 될 때 마다 기록용으로 댓글로 남기겠습니다.

1. 저번 코드리뷰 받은 내용에서 좀 더 궁금했던 점

**Comment by jongfeel** [comment, marked as answer]
네 그렇게 하는게 더 좋을 수 있습니다.
